### PR TITLE
Test reduction surgery 6: Reduced the remaining obob data

### LINKED ISSUE #480: Speed up tests and load multi-tracer/label data in the ci tests
### Inspiration

Tests run very slow and loading all types of example data in the ci yaml script provides a good example to new contributors, despite (or in addition to) the examples in the contrib doc.

### Description

Tests for the execution time of tests and test setup should be added to the code added in PR #506:

https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/ef6db15f72bc6e738e76e6ff7bcbc59d836272a5/DataRepo/tests/tracebase_test_case.py#L24-L43

For the tests that fail:

- Create minimal test data for the tests that currently load the multiple tracer/label example data
- Add the full multiple tracer/label example data to the load in the test yaml file

Also:

- Add the example data load examples to the contrib doc
- Change the example loads in the contrib doc to use load_study only

### Alternatives

none

### Dependencies

This issue cannot be started until the completion of the following issue(s):

- #478

### Comment

The tests should be implemented inside `reportRunTime`.  This comment at the top of that method was removed:

> TODO: When issue #480 is implemented, a test should be added here to ensure each test runs in under some
> reasonable threshold.  There might even be a way to ensure the data setup runs quickly as well.

#### 11/15/2023 Note

I identified the following files in `example_data` that are used in the tests:
```
$ git grep -A 1 example_data | grep "/tests/" | grep '"' | cut -d '"' -f 2,4 | grep -E '/|\.' | perl -e 'while(<>){s/.*\"//;if(m%/$%){chomp}print}' | sort -u
DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/Serum results_cor.csv
DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/TraceBase Animal and Sample Table Templates_AR.xlsx
DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/loading.yaml
DataRepo/example_data/consolidated_tracebase_compound_list.tsv
DataRepo/example_data/data_submission_good/accucor1.xlsx
DataRepo/example_data/data_submission_good/accucor2.xlsx
DataRepo/example_data/data_submission_good/animal_sample_table.xlsx
DataRepo/example_data/data_submission_sample_unkres_acc_good/accucor1.xlsx
DataRepo/example_data/data_submission_sample_unkres_acc_good/accucor2.xlsx
DataRepo/example_data/data_submission_sample_unkres_acc_good/animal_sample_table.xlsx
DataRepo/example_data/obob_animals_table.tsv
DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/6eaafasted1_cor.xlsx
DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/6eaafasted2_cor.xlsx
DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/animal_sample_table.xlsx
DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/bcaafasted_cor.xlsx
DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/loading.yaml
DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/alafasted_cor.xlsx
DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/animal_sample_table.xlsx
DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/glnfasted1_cor.xlsx
DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/glnfasted2_cor.xlsx
DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml
DataRepo/example_data/obob_maven_6eaas_inf.xlsx
DataRepo/example_data/obob_maven_6eaas_inf_corrected.csv
DataRepo/example_data/obob_maven_6eaas_inf_corrected_invalid_syn.csv
DataRepo/example_data/obob_maven_6eaas_inf_corrected_valid_syn.csv
DataRepo/example_data/obob_maven_6eaas_inf_new_researcher_err.xlsx
DataRepo/example_data/obob_maven_6eaas_inf_new_researcher_err2.xlsx
DataRepo/example_data/obob_maven_6eaas_inf_sample_dupe.xlsx
DataRepo/example_data/obob_maven_6eaas_serum.xlsx
DataRepo/example_data/obob_maven_c160_serum.xlsx
DataRepo/example_data/obob_samples_table.tsv
DataRepo/example_data/protocols/T3_protocol.tsv
DataRepo/example_data/protocols/loading.yaml
DataRepo/example_data/protocols/protocols.tsv
DataRepo/example_data/sample_and_animal_tables_headers.yaml
DataRepo/example_data/sample_table_headers.yaml
DataRepo/example_data/serum_lactate_timecourse_treatment.tsv
DataRepo/example_data/serum_lactate_timecourse_treatment_new_researcher.tsv
DataRepo/example_data/small_dataset/small_obob_animal_and_sample_table.xlsx
DataRepo/example_data/small_dataset/small_obob_compounds.tsv
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_blank_sample.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_dupes.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_glucose.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_glucose_2.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_glucose_conflicting.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_lactate.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_req_prefix.xlsx
DataRepo/example_data/small_dataset/small_obob_maven_6eaas_serum.xlsx
DataRepo/example_data/small_dataset/small_obob_sample_table.tsv
DataRepo/example_data/small_dataset/small_obob_sample_table_2ndstudy.tsv
DataRepo/example_data/small_dataset/small_obob_sample_table_serum_only.tsv
DataRepo/example_data/small_dataset/small_obob_study_params.yaml
DataRepo/example_data/small_dataset/small_obob_study_prerequisites.yaml
DataRepo/example_data/small_multitracer_data/6eaafasted1_cor.xlsx
DataRepo/example_data/small_multitracer_data/animal_sample_table.xlsx
DataRepo/example_data/small_multitracer_data/bcaafasted_cor.xlsx
DataRepo/example_data/test_dataframes/loading.yaml
DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor.xlsx
DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor_bad_label.xlsx
DataRepo/example_data/testing_data/accucor_with_multiple_labels/samples.xlsx
DataRepo/example_data/testing_data/animal_sample_table_labeled_elements.xlsx
DataRepo/example_data/testing_data/animal_sample_table_labeled_elements_invalid.xlsx
DataRepo/example_data/testing_data/protocols/protocols_with_errors.tsv
DataRepo/example_data/testing_data/protocols/protocols_with_workarounds.tsv
DataRepo/example_data/testing_data/short_compound_list.tsv
DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_animalid_in_animalsheet.xlsx
DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_animalid_in_samplesheet.xlsx
DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_animalid_in_samplesheet_silent.xlsx
DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_row.xlsx
DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_missing_rqd_vals.xlsx
DataRepo/example_data/testing_data/test_study_1/test_study_1_compounds_dupes.tsv
DataRepo/example_data/testing_data/tissues/tissues_with_errors.tsv
DataRepo/example_data/tissues/loading.yaml
DataRepo/example_data/tissues/tissues.tsv
```
And the following files are used in the github test action (as found in the yaml):
```
DataRepo/example_data/tissues/loading.yaml
DataRepo/example_data/consolidated_tracebase_compound_list.tsv
DataRepo/example_data/obob_sample_table.tsv
DataRepo/example_data/obob_maven_6eaas_inf.xlsx
DataRepo/example_data/obob_maven_6eaas_serum.xlsx
DataRepo/example_data/obob_maven_c160_inf.xlsx
DataRepo/example_data/obob_maven_c160_serum.xlsx
```

-----

### Assumptions

- List of assumptions that the code will not explicitly address/check
- E.g. We will assume input is correct (explaining why there is no validation)

### Requirements

- [ ] 1. List of numbered conditions to be met for the feature
- [ ] 2. E.g. Every column/row must display a value, i.e. cannot be empty
- [ ] 3. Numbers for reference & checkboxes for progress tracking

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected Components

- PR #784
  - [x] `DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/Serum results_cor.csv`
  - [x] `DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/TraceBase Animal and Sample Table Templates_AR.xlsx`
  - [x] `DataRepo/example_data/AsaelR_13C-Valine+PI3Ki_flank-KPC_2021-12_isocorr_CN-corrected/loading.yaml`
  - [x] `DataRepo/example_data/data_submission_good/accucor1.xlsx`
  - [x] `DataRepo/example_data/data_submission_good/accucor2.xlsx`
  - [x] `DataRepo/example_data/data_submission_good/animal_sample_table.xlsx`
  - [x] `DataRepo/example_data/data_submission_sample_unkres_acc_good/accucor1.xlsx`
  - [x] `DataRepo/example_data/data_submission_sample_unkres_acc_good/accucor2.xlsx`
  - [x] `DataRepo/example_data/data_submission_sample_unkres_acc_good/animal_sample_table.xlsx`
  - [x] `DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/6eaafasted1_cor.xlsx`
  - [x] `DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/6eaafasted2_cor.xlsx`
  - [x] `DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/animal_sample_table.xlsx`
  - [x] `DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/bcaafasted_cor.xlsx`
  - [x] `DataRepo/example_data/obob_fasted_ace_glycerol_3hb_citrate_eaa_fa_multiple_tracers/loading.yaml`
- [x] `DataRepo/example_data/consolidated_tracebase_compound_list.tsv`
- [x] `DataRepo/example_data/obob_animals_table.tsv`
- [x] `DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/alafasted_cor.xlsx`
- [x] `DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/animal_sample_table.xlsx`
- [x] `DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/glnfasted1_cor.xlsx`
- [x] `DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/glnfasted2_cor.xlsx`
- [x] `DataRepo/example_data/obob_fasted_glc_lac_gln_ala_multiple_labels/loading.yaml`
- [x] `DataRepo/example_data/obob_maven_6eaas_inf.xlsx`
- [x] `DataRepo/example_data/obob_maven_6eaas_inf_corrected.csv`
- [x] `DataRepo/example_data/obob_maven_6eaas_inf_corrected_invalid_syn.csv`
- [x] `DataRepo/example_data/obob_maven_6eaas_inf_corrected_valid_syn.csv`
- [x] `DataRepo/example_data/obob_maven_6eaas_inf_new_researcher_err.xlsx`
- [x] `DataRepo/example_data/obob_maven_6eaas_inf_new_researcher_err2.xlsx`
- [x] `DataRepo/example_data/obob_maven_6eaas_inf_sample_dupe.xlsx`
- [x] `DataRepo/example_data/obob_maven_6eaas_serum.xlsx`
- [x] `DataRepo/example_data/obob_maven_c160_inf.xlsx`
- [x] `DataRepo/example_data/obob_maven_c160_serum.xlsx`
- [x] `DataRepo/example_data/obob_sample_table.tsv`
- [x] `DataRepo/example_data/obob_samples_table.tsv`
- [x] `DataRepo/example_data/protocols/T3_protocol.tsv`
- [x] `DataRepo/example_data/protocols/loading.yaml`
- [x] `DataRepo/example_data/protocols/protocols.tsv`
- [x] `DataRepo/example_data/sample_and_animal_tables_headers.yaml`
- [x] `DataRepo/example_data/sample_table_headers.yaml`
- [x] `DataRepo/example_data/serum_lactate_timecourse_treatment.tsv`
- [x] `DataRepo/example_data/serum_lactate_timecourse_treatment_new_researcher.tsv`
- [x] `DataRepo/example_data/small_dataset/small_obob_animal_and_sample_table.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_compounds.tsv`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_blank_sample.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_dupes.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_glucose.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_glucose_2.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_glucose_conflicting.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_lactate.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_inf_req_prefix.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_maven_6eaas_serum.xlsx`
- [x] `DataRepo/example_data/small_dataset/small_obob_sample_table.tsv`
- [x] `DataRepo/example_data/small_dataset/small_obob_sample_table_2ndstudy.tsv`
- [x] `DataRepo/example_data/small_dataset/small_obob_sample_table_serum_only.tsv`
- [x] `DataRepo/example_data/small_dataset/small_obob_study_params.yaml`
- [x] `DataRepo/example_data/small_dataset/small_obob_study_prerequisites.yaml`
- [x] `DataRepo/example_data/small_multitracer_data/6eaafasted1_cor.xlsx`
- [x] `DataRepo/example_data/small_multitracer_data/animal_sample_table.xlsx`
- [x] `DataRepo/example_data/small_multitracer_data/bcaafasted_cor.xlsx`
- [x] `DataRepo/example_data/test_dataframes/loading.yaml`
- [x] `DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor.xlsx`
- [x] `DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor_bad_label.xlsx`
- [x] `DataRepo/example_data/testing_data/accucor_with_multiple_labels/samples.xlsx`
- [x] `DataRepo/example_data/testing_data/animal_sample_table_labeled_elements.xlsx`
- [x] `DataRepo/example_data/testing_data/animal_sample_table_labeled_elements_invalid.xlsx`
- [x] `DataRepo/example_data/testing_data/protocols/protocols_with_errors.tsv`
- [x] `DataRepo/example_data/testing_data/protocols/protocols_with_workarounds.tsv`
- [x] `DataRepo/example_data/testing_data/short_compound_list.tsv`
- [x] `DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_animalid_in_animalsheet.xlsx`
- [x] `DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_animalid_in_samplesheet.xlsx`
- [x] `DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_animalid_in_samplesheet_silent.xlsx`
- [x] `DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_empty_row.xlsx`
- [x] `DataRepo/example_data/testing_data/small_obob_animal_and_sample_table_missing_rqd_vals.xlsx`
- [x] `DataRepo/example_data/testing_data/test_study_1/test_study_1_compounds_dupes.tsv`
- [x] `DataRepo/example_data/testing_data/tissues/tissues_with_errors.tsv`
- [x] `DataRepo/example_data/tissues/loading.yaml`
- [x] `DataRepo/example_data/tissues/tissues.tsv`

#### Interface Change description

Describe changes to usage. *E.g. GUI/command-line changes*

#### Code Change Description

Describe code changes planned for the feature. *(Pseudocode encouraged)*

#### Tests

- [ ] 1. A description of at least one test for each requirement above.
- [ ] 2. E.g. Test for req 2 that there's an exception when display value is ''
- [ ] 3. Numbers for reference & checkboxes for progress tracking

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

Final dataset reduction. Tests now run in about 5 minutes.

## Affected Issues/Pull Requests

- Resolves #480

## Review Notes

Many different tests used different data in the dataset, so all had to be updated together.  If one were to do these one test (class) at a time, too much data would be removed, and getting back pieces of it would have been difficult.  So I reviewed all the test that used data from the animals and samples and accucor files and collated all of the compounds (and tracers), tissues, samples, animals, protocols, and peak data that were all used at one point or another and removed everything else.

*This is why doing this reduction one test (class) at a time would have been more laborious and complex.*

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: none
  - Review period: 2 days 
- Associated issue/pull request requirements:
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  - [x] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [ ] Added changes to *Unreleased* section of [`changelog.md`](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

**Comment by hepcat72** [comment]
No more tests remain that run in over 20 seconds.

File: .github/workflows/tracebase-tests.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Load example sample table
         run: |
           python manage.py load_samples \
-            "DataRepo/data/tests/small_obob2/obob_sample_table.tsv"
+          "DataRepo/data/tests/small_obob2/obob_animal_sample_table.tsv"
```
Comment (hepcat72) [inline .github/workflows/tracebase-tests.yml:71]: C. Renamed for clarity. This is an animal sample table. Also adjusted indent for consistency.

File: DataRepo/data/tests/protocols/T3_protocol.tsv
Comment (hepcat72) [inline DataRepo/data/tests/protocols/T3_protocol.tsv:1]: C. Unused file.  Deleted a number of unused files.

File: DataRepo/data/tests/small_obob2/compounds.tsv
Comment (hepcat72) [inline DataRepo/data/tests/small_obob2/compounds.tsv:1]: C. Reduced to just the compounds used.

**Review by lparsons — APPROVED** [review]
The updated dataset names seem better, and the tests are running faster which is great. Nice work.

It looks like you decided to go about this in your own way, which is OK. However, this method means that lots of test data was removed/changed instead of new, smaller datasets being added as needed.

I would have preferred (and I did ask) that tests be updated to use new, smaller copies of data files that were collected in a new "test data" directory. No data would need to have been removed until all tests using it were updated. This would have focused each PR on a single test class.

As long as we

**Comment by hepcat72** [comment]
While we were stopped on our trip back from Buffalo, I shuttled through my emails and was shocked that you said I went my own way and didn't do what you had requested, because I thought that this end result was exactly what you wanted. After being bewildered for awhile, I suddenly realized that there was a miscommunication. When you were talking about breaking things up, I thought we had the same exact end goal in mind (the content of this PR) and that what you were talking about breaking up, was the "work" (I.e. the PRs) of implementing this result. I didn't realize you were talking about breaking up the data into different sub-test directories. I'd thought we were talking about separating the example data from the test data, which is what I did. That's why neither of us was understanding one another! In any case, it's not a big leap to go from this to separate test directories. And in fact, to the best of my knowledge, all the original data is still present in the examples directory, except some test copies that were made, which were vestigial (no longer used in tests). Note, there was another PR you probably didn't see that was merged last week that may have made that apparent.

**Comment by hepcat72** [comment]
> this method means that lots of test data was removed/changed instead of new, smaller datasets being added as needed.

If you can identify any removed/changed data, I will recover it, because I had made it a point in this work to keep all data.  I don't believe any data is removed/missing, but if you are able to identify any, it is a valid issue to fix.  I will merge this and if you are able to identify missing/changed data other than the data added and create an issue, I will recover it.

**Pull request merged**